docs/apache-airflow-providers-cncf-kubernetes/operators.rst

```diff
@@ -82,29 +336,565 @@ equivalent YAML/JSON object spec for the Pod you would like to run.
+  {pod_launcher.py:113} WARNING - Pod not yet started: airflow-pod-operator-env-3824c08cb2f04af7928103a027189668
+  {pod_launcher.py:176} INFO - Event: airflow-pod-operator-env-3824c08cb2f04af7928103a027189668 had an event
+  of type Succeeded
+  {pod_launcher.py:289} INFO - Event with job id airflow-pod-operator-env-3824c08cb2f04af7928103a027189668 Succeeded
+
+  {pod_launcher.py:136} INFO - Reading environment variables TIME_OUT : 5   ENV_TYPE : test
+
+  {pod_launcher.py:176} INFO - Event: airflow-pod-operator-env-3824c08cb2f04af7928103a027189668 had an
+  event of type Succeeded
+  {pod_launcher.py:289} INFO - Event with job id airflow-pod-operator-env-3824c08cb2f04af7928103a027189668 Succeeded
+  {pod_launcher.py:176} INFO - Event: airflow-pod-operator-env-3824c08cb2f04af7928103a027189668 had an event
+  of type Succeeded
+  {pod_launcher.py:289} INFO - Event with job id airflow-pod-operator-env-3824c08cb2f04af7928103a027189668 Succeeded
+  ...................................................................................................................
+
+
+
+Accessing ConfigMap
+===========================
+
+
+- YAML file for creating ConfigMap in Kubernetes
+
+.. code-block:: yaml
+
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: myapp-config
+  data:
+    TIME_OUT: "15"
+    ENV_TYPE: "test"
+
+
+
+- Creating ConfigMap using ``kubectl`` command
+
+.. code-block:: bash
+
+  $ kubectl apply -f k8s_configmap.yaml
+    configmap/myapp-config created
+
+  $ kubectl describe configmaps myapp-config
+    Name:         myapp-config
+    Namespace:    default
+    Labels:       <none>
+    Annotations:  <none>
+
+    Data
+    ====
+    TIME_OUT:
+    ----
+    15
+    ENV_TYPE:
+    ----
+    test
+    Events:  <none>
+
+
+
+- Accessing variables from ConfigMap inside the Pod
+
+.. code-block:: python
+
+  from kubernetes.client import V1ConfigMapEnvSource, V1EnvFromSource
+
+  from airflow import DAG
+  from airflow.providers.cncf.kubernetes.operators.kubernetes_pod import KubernetesPodOperator
+  from airflow.utils.dates import days_ago
+
+  with DAG(dag_id="example_k8s_config_map", start_date=days_ago(1),
+          schedule_interval='@once', tags=["example"]) as dag:
+
+      config_map = [V1EnvFromSource(config_map_ref=V1ConfigMapEnvSource(name='myapp-config')), ]
+
+      task1 = KubernetesPodOperator(task_id='k8s_pod_operator_config_map_task',
+                                    name='airflow_pod_operator_config_map',
+                                    namespace='default',
+                                    image='alpine',
+                                    env_from=config_map,
+                                    cmds=["sh", "-c",
+                                          'echo "Reading environment variables TIME_OUT : $TIME_OUT   ENV_TYPE :'
+                                          ' $ENV_TYPE"'],
+                                    startup_timeout_seconds=60,
+                                    )
```

Comment:
Please move this into its own python file and then reference it, unless @kaxil or @potiuk thinks it's ok to keep small snippets in documentation.